.github/workflows/notify-slack-on-release-merge.yml

```diff
@@ -0,0 +1,75 @@
+name: Slack notifications for releases/* merges
+
+# This CI job will push notifications to Slack whenever code is merged into any releases/* branch
+
+on:
+  push:
+    branches:
+      - 'releases/*'
+
+jobs:
+  notify-slack:
+    name: 'Emit Slack notification(s)'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Parse event to Slack messages
+        env:
+          GITHUB_CONTEXT: ${{ toJson(github) }}
+        # This is a bit of shell kung fu that formats the contents of the GitHub event into a multi-line Slack message
+        # The steps are as follows:
+        # 1. Pull commit data from the Github context and format it as a TSV with jq
+        # 2. Clean up the TSV output so that it's more easily parsed by awk
+        # 3. Use awk to format the TSV into a Slack message
+        # 4. Use sed and tr to deal with new-line escaping issues
+        # 5. Write the output to a file
+        # 6. Use Github action syntax to load the contents of the file into an environment variable
+        run: |
+          echo $GITHUB_CONTEXT | jq '.event.commits[] | [.url, .id[0:7], .author.username, .timestamp, .message] | @tsv' | sed 's/"//g' | sed 's/\\t/;/g' | sed 's/\\n/;/g' | sed 's/\\//g' | awk -F';' '{print "• <"$1"|"$2"> (<https://github.com/"$3"|"$3">, "$4") - "$5}' | sed 's/$/\\n/g' | tr -d '\n' > /tmp/parsed_github_context
+          echo "SLACK_COMMITS=$(cat /tmp/parsed_github_context)" >> $GITHUB_ENV
```

Comment:
Why is this done using an environment variable instead of an output parameter?

eg you could `echo "SLACK_COMMITS=$(cat /tmp/parsed_github_context)" >> $GITHUB_OUTPUT` and then access it as `"${{ steps.parse-slug.outputs.SLACK_COMMITS }}"` so that it is more apparent where it is comming from. I think I'd prefer this method, as it uses localized variables so it seems easier to debug.